src/coordinator/tsdb/remote/codecs.go

```diff
@@ -80,6 +91,15 @@ func DecodeFetchResult(_ context.Context, rpcSeries []*rpc.Series) ([]*ts.Series
 	return tsSeries, nil
 }
 
+func decodeTags(tags []*rpc.Tag) models.Tags {
```

Comment:
why not just have []rpc.Tag ?

Reply:
Required to conform to what protobuf expects